### DISCUSSION TITLE
本科开题报告布局问题

### DISCUSSION THREAD

**Discussion by LZHMS** [note286/xduts#218] [Q&A] [answered]
## 相关需求
如下图所示，在 ”主要研究内容、要解决的问题及本文的初步方案“ 表格中剩下较多的空白区域，希望下一章节 ”工作的主要阶段、进度和完成时间“ 内容可以提前合并进来。
![image](https://github.com/user-attachments/assets/1dcd6370-0979-46f0-8ec9-f59a7c299c42)

## 相关尝试
在xduugtp.clls文件中，尝试在"工作的主要阶段、进度和完成时间"中添加，`\skip_vertical:n { -\l__xdu_box_rule_dim }` 和 `\skip_vertical:n { \z@skip }` 仍然没有作用。
```latex
        { 主要研究内容、要解决的问题及本文的初步方案 }
          {
            \__xdu_bib:
            \end   { tpbox }
            \begin { tpbox } [ tpboxii ]
          }
        { 工作的主要阶段、进度和完成时间 }
          {
            \end   { tpbox }
     % +++  \skip_vertical:n { -\l__xdu_box_rule_dim }
     % +++  \skip_vertical:n { \z@skip }
            \begin { tpbox } [ tpboxi ]
          }
        { 已进行的前期准备工作 }
          {
            \end   { tpbox }
            \skip_vertical:n { -\l__xdu_box_rule_dim }
            \skip_vertical:n { \z@skip }
            \begin { tpbox } [ tpboxii ]
          }
```
## 相关问题
由于个人对latex了解不深，相关配置样式代码不甚明白，前面只是参照其他格式进行的尝试，非常感谢能够提供修改的意见？

**Comment by note286** [comment, marked as answer]
教务处提供的Word模板中，`五、工作的主要阶段、进度和完成时间`就是新开一页。

**Reply by LZHMS** [reply]
好的，明白了。感谢回复。

**Reply by LZHMS** [reply]
再弱弱地问一句，如果想合并能否修改呢？

**Reply by note286** [reply]
可以，但没必要，没人在意，认真写内容即可。
